src/app/tests/suites/certification/Test_TC_LVL_6_1.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 24.6.1. [TC-LVL-6.1] Stop Verification (DUT as Server)
+
+config:
+    cluster: "Level Control"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+
+    - label: "Sending on command"
+      cluster: "On/Off"
+      command: "on"
+
+    - label: "Precondition: DUT level is set to 0x80"
+      command: "Step"
+      arguments:
+          values:
+              - name: "stepMode"
+                value: 0
+              - name: "stepSize"
+                value: 128
+              - name: "transitionTime"
+                value: 20
+              - name: "optionMask"
+                value: 0
+              - name: "optionOverride"
+                value: 0
+
+    - label: "Wait 3000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 3000
+
+    #Disabling as current level value read was returning null on darwin build.
```

Comment:
That's because a previous test did not clean up after itself, presumably.  The right fix is to fix that test, not this.

Reply:
The precondition is changed now i will update according to it bcz we need to set current level to its lowest point